src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -386,7 +386,7 @@ async function initializeForDebugging(context: IActionContext, folder: Workspace
 
 // tslint:disable-next-line: export-name
 export async function scaffoldNetCore(context: ScaffolderContext): Promise<ScaffoldFile[]> {
-    const os = context.os ?? await context.promptForOS();
```

Comment:
An alternative fix would be to not use `context.os` after this point (in favor of the local `os` value), which was likely my intent at the time.